### PR TITLE
Add songrec

### PR DESCRIPTION
**Pull request by sandsmark**
It is a Rust application using Cargo, so harden based on common supply chain attacks seen.

https://github.com/marin-m/SongRec

Also based the config on other players in the repo (mpv etc.).

### CODE REVIEW

File: etc/profile-m-z/songrec.profile [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+include globals.local
+
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-passwdmgr.inc
+include disable-programs.inc
+include disable-shell.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/songrec.profile:16]: maybe add
```
noblacklist ${MUSIC}
include disable-xdg.inc
```

File: etc/profile-m-z/songrec.profile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+include whitelist-common.inc
+include whitelist-player-common.inc
```
Comment (rusty-snake) [inline etc/profile-m-z/songrec.profile:19]: Can we add wusc, wruc, wvc?

File: etc/profile-m-z/songrec.profile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+apparmor
+caps.drop all
+netfilter
+no3d
+nogroups
+nonewprivs
```
Comment (rusty-snake) [inline etc/profile-m-z/songrec.profile:26]: Add `noinput`

File: etc/profile-m-z/songrec.profile [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+noroot
+notv
+nou2f
+novideo
+protocol unix,inet,inet6
+seccomp
+seccomp.block-secondary
+shell none
```
Comment (rusty-snake) [inline etc/profile-m-z/songrec.profile:34]: `tracelog`?

File: etc/profile-m-z/songrec.profile [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+disable-mnt
+private-bin songrec,ffmpeg
+private-dev
```
Comment (rusty-snake) [inline etc/profile-m-z/songrec.profile:38]: can we add `private-cache`?

File: etc/profile-m-z/songrec.profile [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+private-tmp
+
+dbus-user none
+dbus-system none
```
Comment (rusty-snake) [inline etc/profile-m-z/songrec.profile:42]: Can we add `read-only ${HOME}`?

File: etc/profile-m-z/songrec.profile [hunk 7]
```diff
@@ -0,0 +1,42 @@
+# Firejail profile for songrec
+# Description: An open-source Shazam client for Linux
+# This file is overwritten after every install/update
+# Persistent local customizations
+include songrec.local
+# Persistent global definitions
+include globals.local
+
+
```
Comment (rusty-snake) [inline etc/profile-m-z/songrec.profile:9]:
```suggestion
```

File: etc/profile-m-z/songrec.profile [hunk 8]
```diff
@@ -0,0 +1,42 @@
+# Firejail profile for songrec
```
Comment (rusty-snake) [inline etc/profile-m-z/songrec.profile:1]: Add it to firecfg.config

**Comment by smitsohu** [comment]
Merging, thanks for the profile!

I'll add suggestions made by @rusty-snake in a separate commit.

**Pull request merged**